libr/bin/obj.c

```diff
@@ -8,17 +8,135 @@
 	if (binfile->rbin->verbose) \
 	eprintf
 
+static void mem_free(void *data) {
+	RBinMem *mem = (RBinMem *)data;
+	if (mem && mem->mirrors) {
+		mem->mirrors->free = mem_free;
+		r_list_free (mem->mirrors);
+		mem->mirrors = NULL;
```

Comment:
No need to null

Reply:
Are you sure? I'm just keeping the code as it was, I didn't really looked at it, just moved the function from one place to another.